scenario_player/utils/configuration/settings.py

```diff
@@ -40,9 +41,11 @@ class EnvironmentConfig:
     raiden_client: str
     wait_short: int
     wait_long: int
+    development_environment: ContractDevEnvironment = ContractDevEnvironment.DEMO
 
     def __post_init__(self):
         self.eth_rpc_endpoint_iterator = itertools.cycle(self.eth_rpc_endpoints)
+        self.development_environment = ContractDevEnvironment(self.development_environment)
```

Comment:
Is this explicit cast necessary?

Reply:
This allows calling `EnvironmentConfig(**json.load(...))` directly. Without it, you would have to explicitly convert the "unstable" string to `EnvironmentConfig("unstable")`, first. That would work too, but this way the JSON file and the EnvironmentConfig can be seen as the same thing more easily.